main.py

```diff
@@ -23,6 +24,12 @@
 else:
     os.environ['DATABASE_URL'] = f"{os.environ['DATABASE_URL']}{SSLString}"
 
+# for now we just remove the [] from the string later we split to get an array, chose to have [] in the .env to make clear its used as an array
+if os.getenv('GUILD_IDS') is not None and len(os.getenv('GUILD_IDS')) > 2:
```

Comment:
It might be a good idea to only enable this if `DEV_MODE` is set to true

Reply:
DEV_MODE has been removed all together (for now), some improvements have been made to make it more noob friendly and straightforward